compose/networking.md

```diff
@@ -105,43 +106,46 @@ Each service can specify what networks to connect to with the *service-level* `n
 
 Here's an example Compose file defining two custom networks. The `proxy` service is isolated from the `db` service, because they do not share a network in common - only `app` can talk to both.
 
-    version: "3"
-    services:
-
-      proxy:
-        build: ./proxy
-        networks:
-          - frontend
-      app:
-        build: ./app
-        networks:
-          - frontend
-          - backend
-      db:
-        image: postgres
-        networks:
-          - backend
-
-    networks:
-      frontend:
-        # Use a custom driver
-        driver: custom-driver-1
-      backend:
-        # Use a custom driver which takes special options
-        driver: custom-driver-2
-        driver_opts:
-          foo: "1"
-          bar: "2"
+```yaml
+version: "3"
+services:
 
+   proxy:
+    build: ./proxy
+      networks:
+        - frontend
+   app:
+     build: ./app
+      networks:
+        - frontend
+        - backend
+   db:
+     image: postgres
+      networks:
+        - backend
+
+  networks:
+    frontend:
+     # Use a custom driver
+      driver: custom-driver-1
+  backend:
+    # Use a custom driver which takes special options
+    driver: custom-driver-2
+    driver_opts:
+    foo: "1"
+    bar: "2"
```

Comment:
This indentation looks incorrect; `networks:` here is a top-level section, so should not be indented